### PR TITLE
Update commercial_support.rst

### PR DESCRIPTION
**Pull request by northrivergeo**
Hey - Good morning. I'm updating the NRGS "About" section  (lines 424-427) as I've not revisited in this in several years. I also have a new logo - but I'm not sure what to go about that. Can I email it to someone or add it somewhere? Thanks in advance for any help and just in general - thanks.

### CODE REVIEW

File: source/site/forusers/commercial_support.rst [hunk 1]
```diff
@@ -422,9 +422,9 @@ services using the Open Data Kit / KoBo Toolbox software stack.
    :width: 75 px
 
 |northriver| `North River Geographic Systems, Inc <http://www.northrivergeographic.com/>`_
-is a small GIS Consulting firm located in the Southeastern United States. We
-provide Geospatial services such as data conversion, ESRI and FOSS4G software
-support, cartography, and data analysis.
+is a geospatial consulting firm located in the United States. We povide suport for QGIS,
```
Comment (DelazJ) [inline source/site/forusers/commercial_support.rst]:
```suggestion
is a geospatial consulting firm located in the United States. We provide support for QGIS,
```

**Comment by DelazJ** [comment]
For the logo, they are located at https://github.com/rjhale1971/QGIS-Website/tree/patch-1/resources/en/site/about/images so you can drop the existing one and upload the new.

**Comment by northrivergeo** [comment]
Oh man - thanks for fixing the typos. Heh. I was so nervous about doing this all in github I didn't check my spelling.  Will do - I'll get that changed. Thank you for the help!

**Comment by DelazJ** [comment]
> I was so nervous about doing this all in github I didn't check my spelling.

If it can reassure you, there's nothing you can break here, so no worries. You don't have direct access to the official repo (you only have a copy in which you can do EVERYTHING you want), and at worst, you'll only have to fix what went wrong to get it merged :)

**Comment by northrivergeo** [comment]
Hah - Thanks. I still worry about this stuff and I know every precaution has been taken - but "Randy did a merge and the entire project crashed!".

**Pull request merged**